import-scripts/patients_vl.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import numpy as np
+import requests
+
+patients = np.genfromtxt('data/patients.csv', delimiter=',', dtype=None, encoding=None, skip_header=1)
+vl_obs = np.genfromtxt('data/vl_obs.csv', delimiter=',', dtype=None, encoding=None, skip_header=1)
+
+
+# SAVE PERSON OBJECT
```

Comment:
Just as an FYI, OpenMRS REST API will accept a POST to the patient end point like this:

```json
{
    "person": {
        "names": [{
            "givenName": "John",
            "familyName": "Arbuckle"
        }],
        "gender": "M",
        "birthdate": "1970-01-01"
    },
    "identifiers": [{
        "identifier": "AAAAAA",
        "identifierType": "05a29f94-c0ed-11e2-94be-8c13b969e334",
        "location": "58c57d25-8d39-41ab-8422-108a0c277d98",
        "preferred": true
    }]
}
```

Which means we could reduce the number of requests to create a patient down to 1 (from the current 2).

Reply:
Waoh I didn't know this, will do this for sure